rotkehlchen/history/events.py

```diff
@@ -291,14 +291,16 @@ def query_trades(
                 self.actions_per_location['trade'][given_location] = 0
             trades = self.query_location_trades(from_ts, to_ts, Location.EXTERNAL, only_cache)
             # Look for trades that might be imported from CSV files
-            for csv_location in EXTERNAL_EXCHANGES:
+            exchanges_connected_by_credentials = self.exchange_manager.connected_exchanges.keys()
+            connected_locations = self.db.get_connected_locations()
+            csv_locations = set(connected_locations) - set(exchanges_connected_by_credentials)
+            for csv_location in csv_locations:
```

Comment:
Don't change anything here.

I am changing this entire section in my PR. From your perspective it's probably a correct change but I am changing the way this code works here so this becomes redundant and would conflict with my changes.